tests/unit/components/frost-textarea-test.js

```diff
@@ -21,29 +22,193 @@ describeComponent(
+      ).to.be.false
+
+      expect(
+        component.get('disabled'),
+        'disabled: false'
+      ).to.be.false
+
+      expect(
+        component.get('tabindex'),
+        'tabindex: 0'
+      ).to.be.eql(0)
+
+      expect(
+        component.get('cols'),
+        'cols: null'
+      ).to.be.null
+
+      expect(
+        component.get('rows'),
+        'rows: null'
+      ).to.be.null
+
+      expect(
+        component.get('placeholder'),
+        'placeholder: null'
+      ).to.be.null
+
+      expect(
+        component.get('readonly'),
+        'readonly: false'
+      ).to.be.false
+
+      expect(
+        component.get('wrap'),
+        'wrap: soft'
+      ).to.be.eql('soft')
+
+      expect(
+        component.get('form'),
+        'form: null'
+      ).to.be.null
+
+      expect(
+        component.get('value'),
+        'value: null'
+      ).to.be.null
+    })
+
+    it('has the expect Mixins', function () {
+      expect(
+        PropTypeMixin.detect(component),
+        'PropTypeMixin Mixin is present'
+      ).to.be.true
+    })
+
     describe('when onBlur property is omitted', function () {
       beforeEach(function () {
-        run(() => {
-          component.set('onBlur', undefined)
-        })
+        run(() => component.set('onBlur', undefined))
       })
 
       it('does not throw an error when onBlur action is triggered', function () {
-        expect(function () {
-          component.get('actions.onBlur').call(component)
-        }).not.to.throw(Error)
+        expect(
+          function () {
+            component.get('actions.onBlur').call(component)
+            'error not triggered by onBlur()'
+          },
+        ).not.to.throw(Error)
       })
     })
 
-    it('defaults to zero tabindex', function () {
-      expect(component.tabindex).to.equal(0)
-      expect(this.$('textarea').prop('tabindex')).to.equal(0)
+    it('sets dependent keys correctly', function () {
+      const showClearDependentKeys = [
+        'disabled',
+        'readonly',
+        'value'
+      ]
+
+      expect(
+        component.showClear._dependentKeys,
+        'Dependent keys are correct for showClear()'
+      ).to.eql(showClearDependentKeys)
     })
 
-    it('passes tabindex to the underlying field', function () {
-      component.set('tabindex', -1)
-      expect(component.tabindex).to.equal(-1)
-      expect(this.$('textarea').prop('tabindex')).to.equal(-1)
+    describe('"showClear" computed property', function () {
+      it('is set to "true" when disabled=false, value exists and readonly=false', function () {
+        run(() => {
+          component.set('disabled', false)
+          component.set('value', 'value')
+          component.set('readonly', false)
+        })
+
+        expect(
+          component.get('showClear'),
+          'showClear: true'
+        ).to.be.true
+      })
+
+      it('is set to "false" when disabled=true, value exists and readonly=false', function () {
```

Comment:
The false cases would be more readable if they were packaged into a single test with an `or`

i.e. 

```
it('is set to "false" when disabled=true, value doesn't exist or readonly=true', function () {
```